### PR TITLE
Added ban and whitelist feature

### PR DESCRIPTION
**Pull request by greatman**
Vanilla now checks if the player is banned or not in the whitelist.

### CODE REVIEW

**Comment by ZNickq** [comment]
That should be done inside the identification handler imo...

**Comment by greatman** [comment]
Hmm didin't notice that file. Will check it out.

**Comment by greatman** [comment]
@ZNickq Your talking about IdentificationMessageHandler right?

**Comment by greatman** [comment]
Closing this since it's keeping updating with the repo...

**Comment by stephenwilliams** [comment]
@alta189 calls the Vanilla Project Manager to the stage... @NinjaZidane

File: src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java [hunk 1]
```diff
@@ -37,6 +41,35 @@
 	public void handle(Session session, Player player, IdentificationMessage message) {
 		Event event = new PlayerConnectEvent(session, message.getName());
 		session.getGame().getEventManager().callEvent(event);
+		
+		String playerName = player.getName();
+		Server s = (Server) session.getGame();
+		
+		System.out.println("OMG SHU ICI");
```
Comment (Top-Cat) [inline src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java:48]: erm.... what?

Comment (stephenwilliams) [inline src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java:48]: ^ What he said

File: src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java [hunk 2]
```diff
@@ -37,6 +41,35 @@
...
 		Event event = new PlayerConnectEvent(session, message.getName());
 		session.getGame().getEventManager().callEvent(event);
+		
+		String playerName = player.getName();
+		Server s = (Server) session.getGame();
+		
+		System.out.println("OMG SHU ICI");
+		//We check if the player is banned
+		Iterator<Player> bannedPlayerList = s.getBannedPlayers().iterator();
+		while(bannedPlayerList.hasNext())
```
Comment (stephenwilliams) [inline src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java:51]: You need to follow our formatting standards listed in https://github.com/SpoutDev/Vanilla/blob/master/README.md

File: src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java (hunk 1 above)
Comment (ZNickq) [inline src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java:48]: He didn't know the PR gets automatically updated :P

Comment (greatman) [inline src/main/java/org/spout/vanilla/protocol/handler/IdentificationMessageHandler.java:48]: It was a somewhat debug line to know if it went here. :P  Also what @ZNickq said.

**Pull request closed without merging**